cmd/print.go

```diff
@@ -262,3 +275,19 @@ func getNamespaceInCurrentContext() (string, error) {
 
 	return currentNamespace, err
 }
+
+func (pr *PrintRunner) getProviderSpecificConf() map[string]map[string]string {
+	providerSpecificConf := make(map[string]map[string]string)
+	for flagName, value := range pr.providerSpecificConf {
+		provider, found := lo.Find(pr.providers, func(p string) bool { return strings.HasPrefix(flagName, p+"-") })
+		if !found {
+			continue
+		}
+		conf := strings.SplitN(flagName, provider+"-", 2)[1]
```

Comment:
nit: can we change `conf` to something more meaningful? maybe `confName` or even more meaningful - `flagNameWithoutProvider`. These are not great either, so if you come up with something more creative thats better

Also, consider just trimming the prefix -  `strings.TrimPrefix(flagName, provider+"-")`